src/qz/utils/UsbUtilities.java

```diff
@@ -206,22 +206,22 @@ public void run() {
                                     hex.put(UsbUtil.toHexString(b));
                                 }
 
-                                PrintSocketClient.sendStream(session, event.withData("output", hex));
+                                PrintSocketClient.sendStream(session, event.withData("output", hex), usb);
 
                                 try { Thread.sleep(interval); } catch(Exception ignore) {}
                             }
                         }
-                        catch(WebSocketException e) {
-                            usb.setStreaming(false);
-                            log.error("USB stream error", e);
-                        }
-                        catch(DeviceException e) {
+                        catch(WebSocketException | DeviceException e) {
+                            // Calling usb.close() can cause a hard-crash on macOS
```

Comment:
@Vzor- I thought we had gracefully handled this in our upstream PR, but somehow I'm able to crash the app here if I call `usb.close()`.  https://github.com/gary-rowe/hid4java/pull/153/files

I'm leaving this comment in the source code for now as a future warning.

Crashing it's pretty easy, just close the tab while the device is streaming.